<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WiredConnectionWidget</class>
 <widget class="QWidget" name="WiredConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>232</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Restrict to device:</string>
     </property>
     <property name="buddy">
      <cstring>macAddress</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="HwAddrComboBox" name="macAddress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>This option locks this connection to the network device specified by its
permanent MAC address entered here. Example: 00:11:22:33:44:55</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Cloned MAC address:</string>
     </property>
     <property name="buddy">
      <cstring>clonedMacAddress</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KLineEdit" name="clonedMacAddress">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>The MAC address entered here will be used as hardware address for the
network device this connection is activated on. This feature is known as MAC
cloning or spoofing. Example 11:22:33:44:55</string>
       </property>
       <property name="inputMask">
        <string notr="true">HH:HH:HH:HH:HH:HH;_</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRandomMacAddr">
       <property name="text">
        <string>Random</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>MTU:</string>
     </property>
     <property name="buddy">
      <cstring>mtu</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="mtu">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Only transmit packets of the specified size or smaller, breaking larger packets up
into multiple Ethernet frames</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="specialValueText">
      <string comment="MTU:">Automatic</string>
     </property>
     <property name="suffix">
      <string> bytes</string>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="speedLabel">
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="duplexLabel">
     <property name="text">
      <string>Duplex:</string>
     </property>
     <property name="buddy">
      <cstring>duplex</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="KComboBox" name="duplex">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Request that the device use the specified duplex mode. Either &quot;half&quot; or &quot;full&quot;</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Half</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Full</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Link negotiation:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="linkNegotiation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Device link negotiation. If “Manual” is chosen, “Speed” and “Duplex” values will be forced without checking
the device compatibility. If unsure, leave here “Ignore” or pick “Automatic”.</string>
     </property>
     <item>
      <property name="text">
       <string>Ignore</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Link negotiation:">Automatic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Manual</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="speed">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Request that the device use only the specified speed.</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>10 Mb/s</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100 Mb/s</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 Gb/s</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2.5 Gb/s</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5 Gb/s</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10 Gb/s</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>40 Gb/s</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100 Gb/s</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>HwAddrComboBox</class>
   <extends>QComboBox</extends>
   <header>hwaddrcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>macAddress</tabstop>
  <tabstop>clonedMacAddress</tabstop>
  <tabstop>btnRandomMacAddr</tabstop>
  <tabstop>mtu</tabstop>
  <tabstop>duplex</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
